CONTRIBUTING.md

```diff
@@ -1,51 +1,99 @@
-codebase, so we need your permission to use and distribute your code. We also
-need to be sure of various other things—for instance that you'll tell us if you
-know that your code infringes on other people's patents. You don't have to sign
-the CLA until after you've submitted your code for review and a member has
-approved it, but you must do it before we can put your code into our codebase.
-Before you start working on a larger contribution, you should get in touch with
-us first through the issue tracker with your idea so that we can help out and
-possibly guide you. Coordinating up front makes it much easier to avoid
-frustration later on.
-
-### Code reviews
-
-All submissions, including submissions by project members, require review. We
-use Github Pull Requests (PRs) for this purpose. We recommend you read [this guide](
-https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
-if you haven't created a Pull Request before.
+Before we can use your code, please fill out either the individual or corporate Contributor License Agreement (CLA).
+
+  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](https://code.google.com/legal/individual-cla-v1.0.html).
+  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](https://code.google.com/legal/corporate-cla-v1.0.html).
+
+Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
+
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
+
+### How to contribute code
+
+You can follow these steps to submit your code contribution.
 
+#### Step 1. Open an issue
 Before making any changes, we recommend opening an issue (if one doesn't already
 exist) and discussing your proposed changes. This way, we can give you feedback
 and validated the proposed changes. If the changes are minor (simple bug fix
 of documentation fix), then feel free to open a PR without discussion.
 
-### The small print
+#### Step 2. Make code changes
 
-Contributions made by corporations are covered by a different agreement than the
-one above, see the
-[Software Grant and Corporate Contributor License Agreement](https://cla.developers.google.com/about/google-corporate).
+To make code changes, you need to fork the repository.
+You may need to setup a dev environment, run the unit tests, which are introduced in the later sections of this guide.
 
-### Tools needed for development
+#### Step 3. Create a pull request 
+
+Once the code changes are made,
+you can open a pull request from your branch in your fork to the master branch in [keras-team/keras](https://github.com/keras-team/keras)..
```

Comment:
Keep lines short everywhere (but keep links on a single line).

Remove the extra period at the end.